src/foam/lib/csv/CSVOutputter.js

```diff
@@ -8,13 +8,26 @@ foam.INTERFACE({
   package: 'foam.lib.csv',
   name: 'CSVOutputter',
 
+  properties: [
+    {
+      class: 'Boolean',
+      name: 'isFirstRow'
+    }
+  ],
+
```

Comment:
If I didn't move this out, then I would have had to cast the outputter as CSVOutputterImpl - in csvSink. I didn't think that was a good idea, in the future if we change things around we would have to find the cast and adjust.